.ci_support/linux_64_.yaml

```
@@ -1,10 +1,8 @@
-cdt_name:
-- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.9'
+- '3.10'
```

```suggestion
- '3.11'
```
I don’t know what this file is for, but I guess we should put python 3.11 here

---

?

---

I'll try it and see what the robots do, but it seems like 0.5.0 already got published to conda-forge just from this PR (seems not the best)